packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -480,6 +480,7 @@ class ThemeData extends Diagnosticable {
   final Color canvasColor;
 
   /// The foreground color for widgets (knobs, text, overscroll edge effect, etc).
+  /// It is also known as the secondary color.
```

Comment:
This would be an OK place to mention that the theme's [colorScheme] has [ColorScheme.secondary] and [ColorScheme.onSecondary] (a color that contrasts well with the secondary color). New apps may find it convenient to uses the theme's color scheme instead of the original grab bag of color parameters.

Reply:
Oh, that's awesome, I didn't know that [ColorScheme] addressed this. Will make the update